app/Components/CustomTextInput.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { TextInput, StyleSheet } from 'react-native';
+
+export default class CustomTextInput extends React.Component
+{
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return(
+            <TextInput
+                style={styles.textInput}
+                placeholder={this.props.placeholder}
+                placeholderTextColor="#80DED9"
+                secureTextEntry={this.props.secureTextEntry}
+                onFocus={() => this.props.stateCallback(true)}
+                onEndEditing={(event) => {
+                    this.props.stateCallback(false);
+                    this.props.textInputCallback(event.nativeEvent.text)}
+                }
+                >
```

Comment:
You could just close the component, since it doesn't have any content, like this: <TextInput
  style={styles.textInput}
  ....
/>